### PR TITLE
Novaxr (#34)

### PR DESCRIPTION
**Pull request by Andrey1994**
add novaxr

Signed-off-by: Andrey Parfenov <a1994ndrey@gmail.com>

@daniellasry @Fan1117 openbci/novaxr will be our development branch for a while

### CODE REVIEW

**Comment by daniellasry** [comment]
I'm ok with merging this to master before novaXR is finalized, but the diff looks wrong for some reason.

**Comment by Andrey1994** [comment]
Yep and honestly I have no idea why github shows diff wrong

**Comment by Andrey1994** [comment]
fixed merge conflict and now diff looks fine

**Comment by Andrey1994** [comment]
failed appveyour build - it's a build using my appveyour setup there are no credentials to push to aws s3 so it's  ok

File: .travis.yml [hunk 1]
```diff
@@ -63,13 +64,18 @@ script:
   #- cd $TRAVIS_BUILD_DIR && cd java-package/brainflow && mvn package && cd $TRAVIS_BUILD_DIR/tests/java/ && sudo -H javac -classpath $TRAVIS_BUILD_DIR/java-package/brainflow/target/brainflow-jar-with-dependencies.jar BrainFlowTest.java && ls -l . && sudo -H python3 $TRAVIS_BUILD_DIR/emulator/brainflow_emulator/cyton_linux.py java -classpath ".:$TRAVIS_BUILD_DIR/java-package/brainflow/target/brainflow-jar-with-dependencies.jar" BrainFlowTest 0
```
[the remark this answers is not in the document]
Comment (daniellasry) [reply .travis.yml]: it's no longer needed since you added lines 58, 67, and 68

[the remark this answers is not in the document]
Comment (Andrey1994) [reply .travis.yml]: deleted but we still dont have working java test

**Pull request merged**